### ISSUE TITLE
ESRGAN satellite imagery training

### ISSUE THREAD

**Issue by shah0nawaz** [xinntao/BasicSR#215]
HI @xinntao ,

I am training ESRGAN model on satellite imagery, The HR images are 0.5m. I have 20,000 480x480 RGB images. I am training it for scale 4. My training data is not diverse, because I have cropped these images from just 8 large satellite images.

PSNR value does not improves it moves around 14 to 12 

[ 1] it it possible to train this model on 0.5m resolution satellite imagery to achieve good
results.
[2] Dataset size is 20,000 480x480 size images.
Note: I have changed strict = False in def load_network(self, load_path, network, strict=False):
on strict = True it gives me an error.
In the initial iterations the validation images results are comparatively better, means images are in some how comparable to the LR images, but latter the there is no structures in the images. somewhat like smooth plane images with no structures.

Please guide me what I am doing wrong with this setting

**Comment by xinntao** [comment]
1. What does 0.5m resolution mean? 
2. What error does it throw if you set  strict = True
3. Do you want to achieve best PSNR or perceptual quality? If higher PSNR is what you want, do not use perceptual/vgg loss and GAN loss.

**Comment by shah0nawaz** [comment]
@xinntao  sir, Thanks for the response

0.5 m resolution means, I have dataset of  0.5m/pixel resolution of satellite imagery. 
I solved the error of strict = True. actually I wanted to train the ESRGAN on my satellite imagery dataset.  I want good result by training ESRGAN model on satellite imagery. when I set the configuration file, and run the training process the validation results does not improves. I have not changed anything except the configuration file. I only want good results from ESRGAN model. A sir, I you model the quantitative results are measured during training in terms of PSNR value. I need perceptual quality not PSNR value. But as you built model give me the results in terms of PSNR, that is why I put it in my question. Beside the PSNR value, my validation images results does not improves during training.

Please clear my confusion.

**Comment by shah0nawaz** [comment]
@xinntao  I want to achieve best perceptual quality as per you loss function. I studied the paper you published on this model. I do not smooth results.

**Comment by bellenfanttyler** [comment]
@shah0nawaz did you ever find a way to reduce the smoothness of the model output?

**Comment by IwantNewway** [comment]
> HI @xinntao ,
> 
> I am training ESRGAN model on satellite imagery, The HR images are 0.5m. I have 20,000 480x480 RGB images. I am training it for scale 4. My training data is not diverse, because I have cropped these images from just 8 large satellite images.
> 
> PSNR value does not improves it moves around 14 to 12
> 
> [ 1] it it possible to train this model on 0.5m resolution satellite imagery to achieve good results. [2] Dataset size is 20,000 480x480 size images. Note: I have changed strict = False in def load_network(self, load_path, network, strict=False): on strict = True it gives me an error. In the initial iterations the validation images results are comparatively better, means images are in some how comparable to the LR images, but latter the there is no structures in the images. somewhat like smooth plane images with no structures.
> 
> Please guide me what I am doing wrong with this setting

hi, could you tell me after you reconstruct the image, will you put the cut image back? ,if yes ,please tell me how can do? thanks!!!
